lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -414,6 +414,7 @@ sub _prepare_request_data {
     my $msg_type = $req_storage->{msg_type} ||= $req_storage->{method};
 
     my $params       = $self->make_call_params($c, $req_storage);
+    $params->{'logger_context'} = $req_storage->{logger}->get_context() if $req_storage->{logger};
```

Comment:
@aisha-deriv it's public module, this change looks to be very specific to our internal kitchen, any chance we can do it in one of our hooks?

Reply:
Ill look if it can be moved to some other place, if req_storage is already passing as param may be we can do the extraction for the context in RPC.